docs/guides/getting-started-java-spring.md

```diff
@@ -133,9 +133,9 @@ See our documentation on [adding the Spring Zeebe SDK to your project](/apis-too
 </repositories>
 ```
 
-2. Add the following dependency your `pom.xml` file, as a child of the `<dependencies>` element:
+2. Add the following dependency to your `pom.xml` file, as a child of the `<dependencies>` element:
 
-:::warning
+:::caution
```

Comment:
IMO, this caution/warning should be removed in favor of https://github.com/camunda/camunda-8-get-started-spring/pull/9/files - In a nutshell, the `application.properties` must be adjusted to make it work.